CHANGELOG.md

```diff
@@ -1,4 +1,2 @@
-* Remove usage of goog.isFunction.
-* Fix FirebaseUI-web spinner off for RTL languages.
-* Show warning on accountchooser.com becomes universally opt-out.
-* Add unit test to confirm that if accountchooser.com JS dependency fails to load that the sign in flow would continue to work as expected.
\ No newline at end of file
+* Fixes FirebaseUI-web spinner off for RTL languages.
```

Comment:
You didn't fix this. You need to remove the word "off"
* Fixes FirebaseUI-web spinner for RTL languages.